src/pages/questions/index.tsx

```diff
@@ -156,12 +151,15 @@ export default function Questions() {
         <meta name="description" content="Supercharge your interview prep" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
+      <WarningModal
```

Comment:
I think if we want to check for db status, we should do so in the `__app.tsx` file instead of doing this for each individual page

Reply:
right that is a good idea, is everyone ok with blocking access to whole app if only one db is down (im fine with it)